src/main/java/ch/njol/skript/expressions/ExprAngle.java

```diff
@@ -0,0 +1,78 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Angle")
+@Description({
+	"Represents the passed number value in degrees.",
+	"If radians is specified, converts the passed value to degrees. This conversion may not be entirely accurate, " +
+		"due to floating point precision.",
+})
+@Examples({
+	"set {_angle} to 90 degrees",
+	"{_angle} is 90 # true",
+	"180 degrees is pi # true",
+	"pi radians is 180 degrees # true"
+})
+@Since("INSERT VERSION")
+public class ExprAngle extends SimpleExpression<Number> {
+
+	static {
+		Skript.registerExpression(ExprAngle.class, Number.class, ExpressionType.SIMPLE,
+			"%number% [in ]deg[ree][s]",
+			"%number% [in ]rad[ian][s]");
```

Comment:
```suggestion
			"%number% [in] deg[ree][s]",
			"%number% [in] rad[ian][s]");
```
for ease of reading. Also, any reason multiple numbers aren't supported?

Reply:
I think if we support multiple, it should require `in` to do so

Reply:
Agreed, that would help with `1, 200, and pi degrees` instances.
Though that case maybe makes me think singular only is good.